static/custom.css

```diff
@@ -1,7 +1,17 @@
 body { background-color: var(--bs-dark) !important;
        color: var(--bs-light);
 }
-
 .row{
   margin-bottom: 5%;
+}
+
+.footer{
+  display: table;
+  text-align: center;
```

Comment:
@kwsorensen I think it looks a bit better to do this with `left` alignment like everything else on the page. What do you think?
```suggestion
  text-align: left;
```

Reply:
I'm conflicted. I think center looks better on web, worse on mobile.

Reply:
turns out I can have both!